recipes/bids-validator/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "bids-validator" %}
+{% set version = "1.9.9" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bids-validator-{{ version }}.tar.gz
+  sha256: 29d619e6c3a949d51d8bb02f9eb11f0c19df9a5e0fead8532ccd1d50f0bfb680
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6
+    - pip
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - bids_validator
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/bids-standard/bids-validator
+  summary: Validator for the Brain Imaging Data Structure
+  license: MIT
+  license_file:
+    - LICENSE
+
+extra:
+  recipe-maintainers:
+    - effigies
+    - rwblair
```

Comment:
Let's remove Ross for now. I work with him and am happy to add him later if he wants.

```suggestion
```

Reply:
Just some background on why we ask this. There will be some automated emails from GitHub adding folks to the feedstock. Some folks get really mad at us for this so we need to be sure they know what they are agreeing to. Note that, in the feedstock, you can add them without this step. B/c there it is your responsibility to communicate with the maintainers, not the conda-forge core members.